src/stores/notification.js

```diff
@@ -0,0 +1,91 @@
+import { writable } from 'svelte/store';
+
+function createNotifications() {
+  const { subscribe, update } = writable({ notifications: [] });
+
+  const closeNotification = id => {
+    update(prevState => {
+      const notificationIndex = prevState.notifications.findIndex(
+        notification => notification.id === id,
+      );
+      if (notificationIndex !== -1) {
+        const notification = prevState.notifications[notificationIndex];
+
+        clearTimeout(notification.timeoutRef);
+
+        prevState.notifications.splice(notificationIndex, 1);
+      }
+
+      return { ...prevState, notifications: [...prevState.notifications] };
+    });
+  };
+
+  const addNotification = ({ type, message, timeout } = {}) => {
+    update(prevState => {
+      const notification = {
+        id: prevState.notifications.length
+          ? prevState.notifications[prevState.notifications.length - 1].id + 1
+          : 0,
+        message: message || 'Something went wrong',
+        timeout: timeout || 3000,
+        count: 1,
+        timeoutRef: null,
+        ...getNotificationStyle(type),
+      };
+
+      notification.timeoutRef = setTimeout(() => {
+        closeNotification(notification.id);
+      }, notification.timeout);
+
+      const existingNotification = prevState.notifications.findIndex(
+        item => item.message === message,
+      );
+
+      if (existingNotification !== -1) {
+        notification.count = prevState.notifications[existingNotification].count + 1;
+        notification.id = prevState.notifications[existingNotification].id;
+        clearTimeout(prevState.notifications[existingNotification].timeoutRef);
+
+        const notificationsCopy = [...prevState.notifications];
+        notificationsCopy.splice(existingNotification, 1, notification);
+
+        return {
+          ...prevState,
+          notifications: [...notificationsCopy],
+        };
+      }
+
+      return {
+        ...prevState,
+        notifications: [...prevState.notifications, notification],
+      };
+    });
+  };
+
+  return {
+    subscribe,
```

Comment:
How we supposed to use subscribe function at the moment?

Reply:
Svelte store must provide subscribe method otherwise we can not use it. Usage of subscription yo can see in each file that uses store 
![image](https://github.com/user-attachments/assets/1db2b251-c455-4ba6-bce8-6952ea2092ef)

Reply:
Missed this subscription